internal/resources/policy/kind/network/resource/resource_network_policy_test.go

```diff
@@ -0,0 +1,557 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package networkpolicyresource
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/pkg/errors"
+
+	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/authctx"
+	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/client/proxy"
+	policyorganizationmodel "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/policy/organization"
+	policyworkspacemodel "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/policy/workspace"
+	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/policy"
+	policykindNetwork "github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/policy/kind/network"
+	policyoperations "github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/policy/operations"
+	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/policy/scope"
+	testhelper "github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/testing"
+)
+
+const (
+	networkPolicyResource    = policykindNetwork.ResourceName
+	networkPolicyResourceVar = "test_network_policy"
+	networkPolicyNamePrefix  = "tf-ip-test"
```

Comment:
I belive this prefix should be changed to tf-np-test/ 
what does ip stands for ?

Reply:
Done